server/serverHelpers.js

```diff
@@ -0,0 +1,27 @@
+// function pageFetcher(url) {
```

Comment:
Why was this completely commented out and why did you keep it in?

Reply:
Aaaghh I meant to delete that

Reply:
Oh okay, was just curious